src/components/HelpScoutBeacon/useBeaconSuggestions.js

```diff
@@ -1,15 +1,18 @@
 import { useEffect, useCallback, useState } from 'react'
-import suggestions from './suggestions'
 import { staticApps } from '../../static-apps'
 
-const sectionToSuggestions = new Map(suggestions)
+const rawSuggestionsURL =
+  'https://raw.githubusercontent.com/aragon/aragon/master/src/components/HelpScoutBeacon/suggestions.js'
```

Comment:
@AquiGorka What would you think of passing this as an environment variable, rather than hardcoding it?